doc/api/os.md

```diff
@@ -19,9 +19,19 @@ system.
 added: v0.5.0
 -->
 
+    Stability: 0 - Deprecated: Use [`process.arch`][] instead.
+
 Returns the operating system CPU architecture. Possible values are `'x64'`,
 `'arm'` and `'ia32'`. Returns the value of [`process.arch`][].
 
+## os.BYTEORDER
```

Comment:
What's the point of this change? I think it doesn't make sense to change something if it's not really broken